plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024 Marzieh Raoufnezhad <raoufnezhad at gmail.com>
+# Copyright (c) 2024 Maryam Mayabi <mayabi.ahm at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: proxmox_backup_info
+
+short_description: Proxmox Schedule Backup Info  
+
+description: The `proxmox_backup_info` module displays information such as backup times, VM name, VM ID, mode, backup type, and backup schedule using the Proxmox Server API.
```

Comment:
This documentation is later rendered into a specific doc page for this module. No need to refer (to itself):
```suggestion
description: Displays information such as backup times, VM name, VM ID, mode, backup type, and backup schedule using the Proxmox Server API.
```

Reply:
Retrieve, rather than display. Displaying is something we do (or do not) later in the playbook. ;-)